.github/workflows/phpunit-mysql.yml

```diff
@@ -33,8 +33,13 @@ jobs:
         env:
           MYSQL_ROOT_PASSWORD: rootpassword
         options: --health-cmd="mysqladmin ping" --health-interval 5s --health-timeout 2s --health-retries 5
+        commands:
+          - --sql-mode=ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION
```

Comment:
The workflow is not valid. .github/workflows/phpunit-mysql.yml (Line: 36, Col: 9): Unexpected value 'commands'